packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.css

```diff
@@ -0,0 +1,6 @@
+.recommended-badge {
+    background-color: var(--sds-c-badge-color-background, #4fbe75);
```

Comment:
this badge-color-background seems to default to a light grey. While technically this allows users to modify it, given basic badges may be used for entirely other purposes I'm not sure this is the design token we want to use. The best would definitely be if we could set this token value for this component but sadly that isn't GA yet. Maybe we can set this green color and if a user really doesn't want it they can overwrite it with their own custom result template? I think that's what's likely to happen in most cases anyway.
But I'm not sure frankly so if you think it's a good idea to use this token then stick with it.

Reply:
A suggestion: Could we make a badge component?  as per the mockup, there is a featured badge too which is similar to the recommended badge - just the icon and color is different. So wouldn't it be good if we create a badge component that has label and icon as props. so this way the user can select the color / icon they want!. What do you think?

Reply:
Would our component be different enough from the [slds-badge](https://www.lightningdesignsystem.com/components/badges/#site-main-content) to warrant making it?

Reply:
yeah agreed! won't be much of a difference. 
"no badge component" ✅